### PR TITLE
Package

### PR DESCRIPTION
**Pull request by englertr**
works locally! 
I moved everything with the timeseries to the utils (including scrubbing), so that from the outside its just a single function call

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+[tool.poetry]
+name = "connattractor"
+version = "0.1.0"
+description = "Tools for analyzing and quantifying effects of counfounder variables on machine learning model predictions."
```
Comment (spisakt) [inline pyproject.toml:4]: fix deescription

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+numpy = ">=1.21.1"
+pandas = ">=1.3.1"
+scipy = ">=1.7.1"
+statsmodels = ">=0.12.2"
+seaborn = ">=0.11.1"
+scikit-learn = ">=1.3"
+nilearn = ">=0.9.1"
+matplotlib = ">=3.5.2"
+pingouin = ">=0.5.3"
+PyQt5 = ">=5.15.10"
```
Comment (spisakt) [inline pyproject.toml:35]: what do we need pyQt for?

File: connattractor/utils.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    return resources.files('resources')
+def get_MIST_nii_path():
+    base_path = get_base_path()
+    return Path.joinpath(base_path, 'MIST122_relabeled.nii.gz')
+
+def get_MIST_labels_path():
+    base_path = get_base_path()
+    return Path.joinpath(base_path, 'MIST122_relabeled.tsv')
+
+def get_fcHNN_embedding():
```
Comment (spisakt) [inline connattractor/utils.py:18]: reaname it to get_fcHNN_template_embedding()

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+name = "connattractor"
+version = "0.1.0"
+description = "Tools for analyzing and quantifying effects of counfounder variables on machine learning model predictions."
+authors = ["Tamas Spisak <tamas.spisak@uni-due.de>", "Robert Englert <robert.englert@uk-essen.de>"]
+license = "GPL-3.0-or-later"
+repository = "https://github.com/pni-lab/connattractor"
+homepage = "https://pni-lab.github.io/connattractor/"
+readme = 'README.md'
+keywords = [
+    "functional connectivity analysis",
```
Comment (spisakt) [inline pyproject.toml:11]: add "fcHNN", "functional connectome" and "Hopfield"

**Review by spisakt — CHANGES_REQUESTED** [review]
Looks good. I have a few comments.

File: pyproject.toml (hunk 2 above)
Comment (englertr) [reply pyproject.toml]: removed

**Review by spisakt — APPROVED** [review]
Thank you!

**Pull request merged**